doc/rtd/topics/faq.rst

```diff
@@ -200,6 +200,19 @@ systemd, please make sure to include the following logs.
         $ systemctl --failed
 
 
+Autoinstall, Preruncmd, Postruncmd
+==================================
+Since cloud-init ignores top level userdata cloud-config keys, other projects
+such as Juju and subiquity_ autoinstaller use a yaml-formatted config
```

Comment:
It it worth us linking to Juju's model config docs too? https://discourse.charmhub.io/t/model-config-cloudinit-userdata/512

Reply:
+1 to that